src/benchmarks/micro/libraries/System.Diagnostics/Perf_Activity.cs

```diff
@@ -4,183 +4,23 @@
-
-            ActivityLink[] LargeLinkSet = new ActivityLink[1024];
-            for (int i = 0; i < 1024; i++)
-            {
-                LargeLinkSet[i] = new ActivityLink(default);
-            }
-
-            s_ActivityLarge = s_ActivitySource.StartActivity(
-                "TestActivity",
-                ActivityKind.Internal,
-                parentContext: default,
-                tags: LargeTagSet,
-                links: LargeLinkSet);
-
-            for (int i = 0; i < 1024; i++)
-            {
-                s_ActivityLarge.AddEvent(new ActivityEvent($"TestEvent{i}"));
-            }
-
-            s_ActivityLarge.Stop();
-
-            s_ActivityLinkSmall = new ActivityLink(
-                default,
-                new ActivityTagsCollection(
-                    new Dictionary<string, object>
-                    {
-                        ["tag1"] = "string1",
-                        ["tag2"] = 1,
-                        ["tag3"] = "string2",
-                        ["tag4"] = false,
-                    }));
-
-            s_ActivityLinkLarge = new ActivityLink(
-                default,
-                new ActivityTagsCollection(LargeTagSet));
-        }
-
-        [Benchmark]
-        public void EnumerateActivityTagsSmall()
-        {
-            foreach (var _ in s_ActivitySmall.Tags)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityTagsLarge()
-        {
-            foreach (var _ in s_ActivityLarge.Tags)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityTagObjectsSmall()
-        {
-            foreach (var _ in s_ActivitySmall.TagObjects)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityTagObjectsLarge()
-        {
-            foreach (var _ in s_ActivityLarge.TagObjects)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityLinksSmall()
-        {
-            foreach (var _ in s_ActivitySmall.Links)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityLinksLarge()
-        {
-            foreach (var _ in s_ActivityLarge.Links)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityEventsSmall()
-        {
-            foreach (var _ in s_ActivitySmall.Events)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityEventsLarge()
-        {
-            foreach (var _ in s_ActivityLarge.Events)
-            {
-            }
-        }
-
-        [Benchmark]
-        public void EnumerateActivityLinkTagsSmall()
-        {
-            foreach (var _ in s_ActivityLinkSmall.Tags)
-            {
-            }
-        }
+        [Params(ActivityIdFormat.Hierarchical, ActivityIdFormat.W3C)]
+        public ActivityIdFormat IdFormat { get; set; }
```

Comment:
`[Params]` are applied to all the benchmarks in the given type. Since we would like to keep the old type name (and not move all the benchmarks to a different type), you can use `[Arguments]` instead of `[Params]` which would add a new benchmark with two test cases

https://benchmarkdotnet.org/articles/features/parameterization.html#sample-introarguments

it would look like this:

```cs
[Benchmark]
[Arguments(ActivityIdFormat.Hierarchical)]
[Arguments(ActivityIdFormat.W3C)]
public Activity ActivityAllocations(ActivityIdFormat idFormat)
{
    Activity activity = new Activity("TestActivity");
    activity.SetIdFormat(idFormat);
    activity.Start();
    activity.Stop();
    return activity;
}
```

An alternative would be to keep the existing type as is and introduce a new type with a new name and this new benchmark